packages/core/src/components/MediaPlayer/AudioPlayer/AudioPlayerWithInterface.tsx

```diff
@@ -31,6 +31,7 @@ const AudioPlayerWithInterface = React.forwardRef<
       hidePlaybackIcon = false,
       hideDuration = false,
       hideSlider = false,
+      isLooping = false,
```

Comment:
This can just be passed along with `...rest`, since it's not used here.

Reply:
Hmmm, it's pass over to HeadlessAudioPlayer of line 123, not sure about this one

Reply:
`...rest` is also passed into the HeadlessAudioPlayer, so it'll get passed with those as well.